elastio-fargate-mysql-backup/README.md

```diff
@@ -0,0 +1,49 @@
+# AWS RDS MySQL backup by Elastio and AWS Fargate
+
+## Create AIM Role for Elastio Backup
+
+1. Go to [AWS IAM Roles](https://console.aws.amazon.com/iamv2/home#/roles)
+2. Press Create Role button
+3. Select Elastic Container Service Task under Elastic Container Service for AWS service trusted entity and press Next
+![image](https://github.com/elastio/contrib/assets/81738703/0a7050a0-895b-4227-a609-40bb9c6acb24)
+4. Select `ElastioBackupAdmin` permission and press Next
+5. Type ElastioMySQLBackupRole as a role name and press Create role
+
+## Create ECS Task Definition
+
+1. Go to [AWS Elastic Container Service Task definitions](https://console.aws.amazon.com/ecs/v2/task-definitions)
+2. Press Create new task definition
+3. Type ElastioMySQLBackup as a task definition family name
+4. Select ElastioMySQLBackupRole as Task role
+5. Type Elastio-CLI as container name
+6. Paste `public.ecr.aws/elastio-dev/elastio-cli:latest` in container image URI
+7. Expand Docker configuration and paste `sh,-c` in Entry point and following comman in Command:
+```
+apt-get install default-mysql-client -y && mysqldump -h HOST -u USER -P PORT -p'PASSWORD' DATABASE | elastio stream backup --stream-name MySQL-Elastio-backup
+```
+For example:
+```
+apt-get install default-mysql-client -y && \
```

Comment:
It's not a good idea to hard-code the password like this.  We should be instructing customers in the use of best practices.

The password should be stored in an SSM secret.  The Fargate task config should set an env var from that secret.  And the command line should pass the env var.

Reply:
Added secrets manager.